### PR TITLE
Adding the beego framework

### PR DESCRIPTION
**Pull request by brotherthegrest**
just adding the Beego framework
just doing my part

### CODE REVIEW

**Comment by brotherthegrest** [comment]
Hope this is good enough

**Comment by brotherthegrest** [comment]
give me a second to fix conflicts

File: cmd/template/beegoRoutes.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	"fmt"
+)
+
+func main() {
+	fmt.println("Running at localhost:8080")
+	web.Run()
+}
+
+`)
+}
```
Comment (Melkeydev) [inline cmd/template/beegoRoutes.go:29]: i dont think this is going to work.
This looks like it is actually spinning up a server instance,
and not actually declaring any routes

**Comment by Melkeydev** [comment]
This is straight up a hack.
Beego is some bull shit scam framework.
Do NOT Add this

**Comment by brotherthegrest** [comment]
ok

**Comment by brotherthegrest** [comment]
bro

**Comment by brotherthegrest** [comment]
uncool

**Comment by brotherthegrest** [comment]
Like man I didn't know that a go "open source" framework could be a scam

File: cmd/template/beegoRoutes.go (hunk 1 above)
Comment (brotherthegrest) [reply cmd/template/beegoRoutes.go:29]: Yeah my mistake I misunderstood the task

**Comment by brotherthegrest** [comment]
No intention of hacking or malware like that shit puts me in jail

**Pull request closed without merging**